llama_stack/providers/tests/test_config_helper.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import pytest
+import yaml
+from pydantic import BaseModel, Field
+
+
+@dataclass
+class APITestConfig(BaseModel):
+
+    class Fixtures(BaseModel):
+        # provider fixtures can be either a mark or a dictionary of api -> providers
+        provider_fixtures: List[Dict[str, str]] = Field(default_factory=list)
+        inference_models: List[str] = Field(default_factory=list)
+        safety_shield: Optional[str] = Field(default_factory=None)
+        embedding_model: Optional[str] = Field(default_factory=None)
+
+    fixtures: Fixtures
+    tests: List[str] = Field(default_factory=list)
+
+    # test name format should be <relative_path.py>::<test_name>
+
+
+class TestConfig(BaseModel):
+
```

Comment:
there's a newline here which looks odd -- looks like your editor is not quite set up right for formatting? might be good to get that fixed